.github/ISSUE_TEMPLATE/behavior-bug---plugin-incompatibility.yml

```diff
@@ -0,0 +1,60 @@
+name: Behavior Bug / Plugin Incompatibility
+about: Server Bug or Plugin Incompatibility
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Before reporting a crash here, please make sure you are on the latest version of Paper.
+        Forks of Paper receive no support here. If you are using a fork, please make sure that this crash also happens when using Paper.
+
+  - type: textarea
+    attributes:
+      label: Expected behavior
+      description: What you expected to see
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Observed/Actual behavior
+      description: What you actually saw
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps/models to reproduce
+      description: This may include a build schematic, a video, or detailed instructions to help reconstruct the issue
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Plugin list
+      description: All plugins running on the server.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Paper version.
+      description: |
+        Paste the full, unmodified, output of `/version` on your server WITH the Minecraft version.
+        `latest` is not a version; we require the output of `/version` so we can properly track down the issue
+    validations:
+      required: false
+
+  - type: checkboxes
+    attributes:
+      label: Agreements
+      description: Please confirm the following
+      options:
+        - label: I am running the latest version of Paper from https://papermc.io/downloads
+          required: true
+        - label: I have searched for and ensured there isn't already an open issue regarding this
+          required: true
+        - label: My version of Minecraft is supported by Paper
```

Comment:
People who run unsupported versions will just check it; not really worth having.

Reply:
meh, I still feel like it at least makes ppl think about it for a second

Reply:
See https://canary.discord.com/channels/289587909051416579/746078378108518482/814663631278047284